control/controlhttp/client.go

```diff
@@ -40,11 +40,19 @@ import (
 	"tailscale.com/types/key"
 )
 
-// upgradeHeader is the value of the Upgrade HTTP header used to
-// indicate the Tailscale control protocol.
 const (
-	upgradeHeaderValue  = "tailscale-control-protocol"
+	// upgradeHeader is the value of the Upgrade HTTP header used to
+	// indicate the Tailscale control protocol.
+	upgradeHeaderValue = "tailscale-control-protocol"
```

Comment:
Any thoughts on varying this header value, rather than the switch URL?

I have no strong feelings, just occurred to me that there's slight redundancy between the URL and the header now, if the URL also specifies the exact protocol being switched to.